apps/console/src/auth.html

```diff
@@ -224,6 +226,28 @@
                         return serverOrigin + getTenantPath(tenantDomain);
                     }
 
+                    /**
+                     * Get the ADU API path.
+                     * ADU: Asgardeo Deployment Unit.
+                     * 
+                     * @param {string} path - Path to be appended to the API path.
+                     * @param {string} tenantDomain - Tenant domain.
+                     * @returns {string} Constructed API path.
+                     */
+                    function getADUApiPath(path, tenantDomain) {
```

Comment:
shall we keep this generic (such as `getDeploymentUnitApiPath`), without mentioning product names in open source repo